Module-9/Homework9/PicProwlr/Services/PexelsService.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  PexelsService.swift
+//  Created by Mark Renaud (2024).
+//
+
+import SwiftUI
+
+enum PexelsError: Error {
+    /// Unable to find `PEXELS_API_KEY` environment variable.
+    case missingAPIToken
+    /// Unexpected HTTP Response
+    case httpResponse(code: Int)
+}
+
+class PexelsService {
+    enum HTTPMethod: String {
+        case get = "GET"
+    }
+    
+    /// private let token: String?
+    private let baseURL = Constants.External.pexelsAPIBase
```

Comment:
Interview: How would you handle staging and production URLs? How to easily switch between them?

Reply:
This can be achieved by creating different Xcode Configuration Files for Staging and Production. This approach makes changing endpoints easy by selecting which Config file maps to a particular build configuration.

I have added example .xcconfig files to the project and mapped them to the Debug and Release build schemes, respectively (although the API endpoint is currently the same for both Production and Staging).

Issue #23 - Fixed with 9b3e61e